akd/src/append_only_zks.rs

```diff
@@ -676,7 +677,7 @@ impl Azks {
                 .collect();
         }
 
-        info!("Preload of tree ({} nodes) completed", load_count);
+        debug!("Preload of tree ({} nodes) completed", load_count);
```

Comment:
I'm moving this to `Debug`, as it's a bit noisy considering that it's in the hot path of read oriented APIs. Under load, this may be a source of spammy log messages which offer little value in terms of information.